Timer/force-app/main/default/lwc/timerLWC/timerLWC.html

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div
+    style="
+      background: white;
+      height: 450px;
+      width: 600px;
+      padding: 40px;
+      border-radius: 10px;
+    "
+  >
+    <div style="font-size: 40px; text-align: center; height: 100px">
```

Comment:
Make this a separate component.
This component should not be responsible to hold the state that we need to show hours, minutes, seconds, or not.
Create a separate component that handles this logic and give it a good name.

Reply:
The component should receive hours, minutes and seconds and based on that decide everything.